querybook/webapp/redux/dataDoc/action.ts

```diff
@@ -621,3 +624,43 @@ export function rejectDataDocAccessRequest(
         }
     };
 }
+
+export function getDataDocWithSchema({
+    paginationPage,
+    paginationPageSize,
+    paginationFilter,
+}: {
+    paginationPage?: number;
+    paginationPageSize?: number;
+    paginationFilter?: [{ value: string }];
+}): ThunkResult<Promise<ScheduledDocType[]>> {
+    return async (dispatch, getState) => {
+        const env = getState().environment.currentEnvironmentId;
+        const schemaDataDoc = getState().dataDoc.dataDocWithSchema;
+        const page = paginationPage ?? schemaDataDoc.page;
+        const filtered = paginationFilter ?? schemaDataDoc.filtered;
+        const pageSize = paginationPageSize ?? schemaDataDoc.pageSize;
```

Comment:
the pagination would mess up if pageSize changes?

Reply:
No, the pagination works correctly even if we change the pageSize, I'm going to make a video when I will show it.